### ISSUE TITLE
Emmit

### ISSUE THREAD

**Issue by CarlosSb** [Microsoft/vscode#60893]
- Extension Name: emmet
- Extension Version: 1.0.0
- OS Version: Windows_NT x64 10.0.17134
- VSCode version: 1.28.0

```json 
{
	"activationTimes": {
		"startup": true,
		"codeLoadingTime": 231,
		"activateCallTime": 4,
		"activateResolvedTime": 0,
		"activationEvent": "*"
	}
}
```

**Comment by atilkan** [comment]
very experimental.

**Comment by ramya-rao-a** [comment]
This is mostly related to #60318 and should be fixed in 1.28.1
